CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - The removal of `/v1/report` APIs in this release, that was communicated in release 4.0.0, has been postponed by one release. Consequently, the `/v1/report` APIs will continue to be available in this release.
 - The `/api/docs/swagger` API previously required read on the resource `Integration`.
   Now it only requires users to be authenticated to via the API docs.
+- ROX-18173: A few public endpoints will soon require authentication: `/v1/availableauthproviders`, `/v1/config/public`, `/v1/featureflags`, `/v1/internal/dev/certcheck`, `/v1/internal/dev/envvars`, `/v1/internal/dev/random`, `/v1/internal/dev/reconciliationstats`, `/v1/internal/dev/replicate` and `/v1/resources`. Ensure the flows targeting these endpoints are authenticated.
```

Comment:
nit: make this a multi-line listing of the endpoints?

Also, the remediation can be re-written to `Ensure that any flow interacting with these endpoints is authenticated going forward`.